src/test/unit/project-converter_test.ts

```diff
@@ -78,26 +78,33 @@ suite('AnalysisConverter', () => {
         bowerPackageName = 'some-package',
         packageType = 'element',
         expectedWarnings = [],
+        entrypoints = new Map(),
       } = partialOptions;
+
+      if (!entrypoints.has('some-package')) {
+        entrypoints.set('some-package', ['test.html']);
```

Comment:
(from offline) Move this into the default for `entrypoints` above. Users should provide all of this map if they intend to provide any of it.

Reply:
done